src/locales/en_US.ts

```diff
@@ -1141,6 +1141,18 @@ const strings = {
   stake_error_insufficient_s: 'Insufficient %s',
   stake_error_stake_below_minimum: 'Stake amount below minimum',
   stake_error_unstake_below_minimum: 'Unstake amount below minimum',
+  stake_estimated_staking_fee: 'Estimated Staking Fee',
+  stake_estimated_unstaking_fee: 'Estimated Unstaking Fee',
+  stake_staking_fee_message: 'The specified Staking Fee will be deducted from the amount deposited reducing the amount staked.',
+  stake_unstaking_fee_message: 'The specified Unstaking Fee will be deducted from the amount withdrawn reducing the amount received.',
+  stake_future_unstaking_fee: 'Future Unstake Fee Estimate',
+  stake_future_unstaking_fee_message:
+    'When you unstake these funds, you will incur the estimated unstake fee specified. This amount is only an estimate and is based on the funds currently staked in the pool. The actual fees incurred may be different at the time you unstake.',
+  stake_break_even_time: 'Time to Break Even from Fees',
+  stake_break_even_time_message:
+    'Based on the total fees incurred to stake and unstake your requested amount, this is the estimated amount of time you need to keep funds staked to earn enough rewards to pay for the fees incurred. This time frame is only an estimate as is subject to change based on change in rewards APY and the total amount of funds in the staking pool.',
+  stake_break_even_days: '%1$s days',
+  stake_break_even_days_months: '%1$s days (%2$s months)',
```

Comment:
All strings with parameters should follow our conventions of having an `_s` suffix.

Reply:
👍